index.js

```diff
@@ -30,7 +31,12 @@ Gulp.prototype.watch = function(glob, opt, task) {
     fn = this.parallel(task);
   }
 
-  return vfs.watch(glob, opt, fn);
+  var watcher = chokidar.watch(glob, opt);
+  if (fn) {
+    watcher.on('all', fn);
```

Comment:
Take a look at [my original PR to glob-watcher](https://github.com/wearefractal/glob-watcher/pull/13/files) for a bit of reference. Although we are not going to want to keep re-mapping the divergent event names.

Unless someone can think of a reason why gulp users would want the dir events?